From: OneKVM Project <223006751+lowtech-guy@users.noreply.github.com>
Subject: [PATCH] libraries: leave stripping to the packager

Do not strip shared libraries during compilation.  OpenEmbedded splits debug
symbols and strips runtime objects during packaging.

Upstream-Status: Inappropriate [OpenEmbedded packaging policy]

---
 modules/bin/Makefile                    | 1 -
 modules/isp/cv181x/isp_bin/Makefile     | 1 -
 2 files changed, 2 deletions(-)

diff --git a/modules/bin/Makefile b/modules/bin/Makefile
index b4dd0b7..90f0203 100644
--- a/modules/bin/Makefile
+++ b/modules/bin/Makefile
@@ -57,7 +57,6 @@ $(TARGET_A): $(OBJS)
 $(TARGET_SO): $(OBJS)
 	@$(LD) $(LDFLAGS) $(EXTRA_LDFLAGS) -o $(TARGET_SO) --start-group $(OBJS) $(LIBS) --end-group
 	@echo -e $(GREEN)[LINK]$(END)[$(notdir $(LD))] $(notdir $(TARGET_SO))
-	@$(STRIP) $(TARGET_SO)
 
 clean:
 	@rm -rf $(TARGET_A) $(TARGET_SO)
diff --git a/modules/isp/cv181x/isp_bin/Makefile b/modules/isp/cv181x/isp_bin/Makefile
index a6b7fcb..0cb0f45 100644
--- a/modules/isp/cv181x/isp_bin/Makefile
+++ b/modules/isp/cv181x/isp_bin/Makefile
@@ -70,7 +70,6 @@ $(TARGET_A): $(OBJS)
 $(TARGET_SO): $(OBJS)
 	@$(LD) $(LDFLAGS) $(EXTRA_LDFLAGS) -o $@ --start-group $(OBJS) $(TMP_FOLDER_LIB)/*.o --end-group
 	@echo -e $(GREEN)[LINK]$(END)[$(notdir $(LD))] $(notdir $(TARGET_SO))
-	@$(STRIP) $(TARGET_SO)
 
 sdk_release:
 	-@rm -rf $(ODIR)
-- 
2.50.1
